lib/mongoose_push_web/schemas/request/send_notification/deep/data.ex

```diff
@@ -0,0 +1,28 @@
+defmodule MongoosePushWeb.Schemas.Request.SendNotification.Deep.Data do
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "Request.SendNotification.Deep.Data",
+    description: "In this request one can pass data field only.",
+    type: :object,
+    allOf: [MongoosePushWeb.Schemas.Request.SendNotification.Deep],
+    properties: %{
+      data: %Schema{
+        type: :object,
+        description:
+          "Custom key-values pairs of the message's payload. " <>
+            "The FCM request with nested data can end up with error."
+      }
+    },
+    required: [:data],
+    example: %{
+      "service" => "apns",
```

Comment:
This property is not defined for this schema.

Reply:
Corrected.